### DISCUSSION TITLE
excelente plugin solo una consulta

### DISCUSSION THREAD

**Discussion by screpone** [whooohq/whq-woocommerce-chilexpress-shipping#213] [Support]
Estimado primero que todo hacerle llegar mis felicitaciones por el plugin!! lo segungo tengo una consulta estoy usando actualmente en un marketplace varios vendedores con dokan pero llevo dias tratando de especificar sucursal de origen de envio para cada vendedor y que pueda calcular el envio dependiendo de la ciudad de origen, en el plugin solo veo una opcion de elegir sucursal de origen, saludos y de ante mano muchusimas gracias.

**Comment by EstebanForge** [comment]
Estimado,

https://github.com/whooohq/whq-woocommerce-chilexpress-shipping/issues/97
